src/Movie-Card/Movie-Card.css

```diff
@@ -0,0 +1,75 @@
+.movie-card {
+  height: 90%;
```

Comment:
```suggestion
  height: 90%;
  display: flex;
   flex-direction: column;
   justify-content: flex-start;
   align-items: center;
```

Reply:
I think the flex box will respond better on the parent element, it will align all the children within it